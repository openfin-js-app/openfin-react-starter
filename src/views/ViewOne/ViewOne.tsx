import * as React from 'react';

export default class ViewOne extends React.Component<any,any>{
    render(){
        return(
            <React.Fragment>
                <span>Viewone works</span>
            </React.Fragment>
        );
    }
}