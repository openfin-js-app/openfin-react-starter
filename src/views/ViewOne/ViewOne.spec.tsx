import * as React from 'react';
import { createShallow, createMount } from '@material-ui/core/test-utils';

import ViewOne from './ViewOne';

describe('ViewOne',()=>{

    let shallow;

    beforeAll(() => {
        shallow = createShallow();
    });

    it ('render correctly',()=>{

        const wrapper = shallow(<ViewOne/>);
        const spanEle = wrapper.find('span');

        expect(spanEle).toHaveLength(1);

        expect(spanEle.get(0).props).toMatchSnapshot();

    })
});