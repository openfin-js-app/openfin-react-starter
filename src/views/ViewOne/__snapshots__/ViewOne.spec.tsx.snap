// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewOne render correctly 1`] = `
Array [
  Object {
    "payload": Object {
      "count": 1,
    },
    "type": "CLIENT_SET_VALUE",
  },
  Object {
    "payload": Object {
      "count": -1,
    },
    "type": "CLIENT_SET_VALUE",
  },
]
`;
